voidcrew/edits/mobs/new_player.dm

```diff
@@ -79,19 +79,18 @@
 /**
  * Latejoining
  */
-/mob/dead/new_player/AttemptLateSpawn(rank, obj/structure/overmap/ship/joined_ship)
+/mob/dead/new_player/AttemptLateSpawn(datum/job/job, obj/structure/overmap/ship/joined_ship)
```

Comment:
TG master's args currently is ``(rank)``, would it be better to try to stick to their code as much as possible?